doc/source/local.md

```diff
@@ -94,31 +154,129 @@ Edit the `.env` file with the necessary credentials and settings.
 # Copy this file to .env and replace the credentials with
 # your own before starting the app.
 
+#----------------------------------------------------------
+# IBM Cloud
+#
+# If your services are running on IBM Cloud,
+# uncomment and configure these.
+# Remove or comment out the IBM Cloud Pak for Data sections.
+#----------------------------------------------------------
+
 # Watson Assistant
-ASSISTANT_APIKEY=zzZzzABCsU8DBrvi123HLZwVyHbRlBFf_97n9O123ABC
-ASSISTANT_URL=https://gateway.watsonplatform.net/assistant/api
+CONVERSATION_AUTH_TYPE=iam
+CONVERSATION_APIKEY=zzZzzABCsU8DBrvi123HLZwVyHbRlBFf_97n9O123ABC
+CONVERSATION_URL=https://gateway.watsonplatform.net/assistant/api
+# Optionally, use a non-default skill by specifying your own workspace ID.
+WORKSPACE_ID=<add_assistant_workspace_id>
```

Comment:
should change WORKSPACE_ID to SKILL_ID but that can be done whenever.

Reply:
will do this next PR